pybossa/exporter/export_helpers.py

```diff
@@ -213,3 +213,18 @@ def browse_tasks_export_count(obj, project_id, expanded, filters):
         return
     return session.execute(
             sql, dict(project_id=project_id, **filter_params)).scalar()
+
+
+def filter_table_headers(headers, accepted_task_info_fields):
+            new_headers = []
+            accepted_task_info_fields_set = set(accepted_task_info_fields)
+            for header in headers:
+                if header == 'task__info':
+                    pass
```

Comment:
pass can be avoided w/ flipping the condition

Reply:
if we switch to 
```
if header != 'task__info':
 ...
else:  
     new_headers.append(header) 
```
`task__info` will always get added to `new_headers`.

Reply:
For clarity, there are 3 cases for each header:
1. header = 'task__info', so we skip it
2. header is like 'task__info__[fieldname], so we check fieldname against accepted fields set and add accordingly
3. header is something else, so we add it to the new_headers list (task__id, task__state, etc)

Reply:
Likely due to small tab space, it was not clear that else part of `if header != 'task__info':` won't be adding `new_header` always.